ui/pages/confirmations/components/confirm/info/set-approval-for-all-info/revoke-set-approval-for-all-static-simulation/revoke-set-approval-for-all-static-simulation.tsx

```diff
@@ -0,0 +1,58 @@
+import { NameType } from '@metamask/name-controller';
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React from 'react';
+import { ConfirmInfoRow } from '../../../../../../../components/app/confirm/info/row';
+import Name from '../../../../../../../components/app/name';
+import { Box } from '../../../../../../../components/component-library';
+import {
+  AlignItems,
+  Display,
+} from '../../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../../hooks/useI18nContext';
+import { useConfirmContext } from '../../../../../context/confirm';
+import StaticSimulation from '../../shared/static-simulation/static-simulation';
+
+export const RevokeSetApprovalForAllStaticSimulation = ({
+  spender,
+}: {
+  spender: string;
+}) => {
+  const t = useI18nContext();
+
+  const { currentConfirmation: transactionMeta } = useConfirmContext() as {
+    currentConfirmation: TransactionMeta;
+  };
+
+  const RevokeSetApprovalForAllRows = (
+    <>
+      <ConfirmInfoRow label={t('nfts')}>
```

Comment:
Minor, could we define each row as a separate component in this file for readability?